definitions-tests/definitions_test.go

```diff
@@ -0,0 +1,37 @@
+package definitions_tests
+
+import (
+	"cuelang.org/go/cue"
+	"cuelang.org/go/cue/cuecontext"
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func TestTraits(t *testing.T) {
+	cuectx := cuecontext.New()
+	// parse input
+	inputBytes, err := os.ReadFile("./expose/input.cue")
```

Comment:
I think it's better to make the directory like this:

```
inputs/*.cue
outputs/*.cue
```

The CUE files in inputs/outputs with the same name will be used as a test case.

Reply:
Now I merge outputs and inputs into one `.cue` file, it's more convenient to manage massive test cases.